ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -116,12 +116,12 @@ const statusIconTextMap: { [key in ComplianceCheckStatus]: ClusterStatusObject }
     [ComplianceCheckStatusEnum.PASS]: {
         icon: (
             <Icon>
-                <CheckCircleIcon color="var(--pf-v5-global--success-color--100)" />
+                <CheckCircleIcon color="var(--pf-v5-global--primary-color--100)" />
             </Icon>
         ),
         statusText: 'Pass',
         tooltipText: 'Check was successful',
-        color: 'green',
+        color: 'blue',
```

Comment:
Aha, maybe I missed this in earlier reviews.

The change and the question above by Saif directed my attention to color text. Except green text color of fixable links in Vulnerability Management 1.0, the accessibility. effort removed most text color on classic pages, and as far as I have noticed, PatternFly pages do not usually match text color to icon color.

If this is a new pattern, it seems like a timely subject for discussion at weekly meeting.

Reply:
Good idea and yeah I'm not a fan of color either. Just to make things clear, the `color` property is only used for label color as of now.

Reply:
Might be better off renaming it to make that more obvious

Reply:
Aha, blue text in **Compliance status** column is because link variant?

Reply:
Yep! I guess we changed the icon color too, but that one is at least more obvious